app/controllers/ApiController.scala

```diff
@@ -90,6 +89,41 @@ class ApiController @Inject()(authActionFactory: AuthActionFactory,
     Ok(Json.toJson(searchManagementRepository.listAllInputTags()))
   }
 
+  def listAllUsers(): Action[AnyContent] = authActionFactory.getAuthenticatedAction(Action) {
+    Ok(Json.toJson(searchManagementRepository.listAllUsers()))
+  }
+
+  def getUser(userId: String): Action[AnyContent] = authActionFactory.getAuthenticatedAction(Action) {
+    Ok(Json.toJson(searchManagementRepository.getUserById(userId)))
+  }
+
+  def lookup(by: String, value: String): Action[AnyContent] = authActionFactory.getAuthenticatedAction(Action) {
```

Comment:
could this be getUser() with a different params?   Or, call it lookupUser???

Reply:
2 methods added for lookup of user: via username and email